charts/alerts/templates/deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
             imagePullSecrets:
             containers:
                 - name: {{.Chart.Name}}
-                  image: ghcr.io/cdcgov/phdi/alerts:{{ .Values.image.tag }}
+                  image: ghcr.io/cdcgov/phdi/{{.Values.name}}:{{ .Values.image.tag }}
```

Comment:
I think in order to get all these changes to publish, you'll need to change the `version` in each `Chart.yaml` file - but @nickbristow or @KennethSkylight would know for sure, they've been changing these files much more recently.

Reply:
Just confirmed with @KennethSkylight that it needs to be changed and made the appropriate changes!